omnibus/files/private-chef-cookbooks/private-chef/recipes/elasticsearch.rb

```diff
@@ -159,7 +159,16 @@
 end
 
 component_runit_service 'elasticsearch' do
-  action [:enable, :start]
+  action [:enable]
+end
+
+# While Restoring the chef server backup, enable is always successful 
+# but start is not which is the root cause of restore issue.
+# So It is moved to separate resource for retrying start incase of failure 
+component_runit_service 'elasticsearch' do
+  action [:start]
```

Comment:
We could actually use the the start and enable actions with the retries and keep it a single resource, which avoids some potential backend confusion in the client since both resources have the same name

Reply:
yes @tas50, That's true.